.vib/kuberay-apiserver/goss/vars.yaml

```diff
@@ -0,0 +1,6 @@
+binaries:
+  - kuberay-apiserver
+root_dir: /opt/bitnami
+directories:
```

Comment:
You are adding this vars but not loading the `check-directories.yaml` Goss' template in `goss.yaml`